Source/WebCore/platform/graphics/TrackPrivateBaseClient.h

```diff
@@ -36,7 +36,12 @@ using TrackID = uint64_t;
 
 class TrackPrivateBaseClient : public CanMakeWeakPtr<TrackPrivateBaseClient> {
 public:
+    using Task = Function<void()>;
+    using Dispatcher = Function<void(Task&&)>;
+
     virtual ~TrackPrivateBaseClient() = default;
+    enum Type { Text, Audio, Video };
```

Comment:
Please only use enum classes in new code, and specify their underlying type (here uint8_t).